src/System Application/App/Email/src/Message/EmailMessageImpl.Codeunit.al

```diff
@@ -53,11 +53,9 @@ codeunit 8905 "Email Message Impl."
     end;
 
     procedure Create(ToRecipients: Text; Subject: Text; Body: Text; HtmlFormatted: Boolean)
-    var
-        EmptyList: List of [Text];
     begin
 #pragma warning disable AA0205
-        Create(EmptyList, Subject, Body, HtmlFormatted, false);
+        Create(ToRecipients, Subject, Body, HtmlFormatted, false);
```

Comment:
AA0205:
Variables must be initialized before usage.

No more variable so why is the pragma still here?